game/game/movealgo.cpp

```diff
@@ -708,6 +708,20 @@ void MoveAlgo::startDive() {
     }
   }
 
+void MoveAlgo::invalidatePhysics() {
```

Comment:
This function doesn't do what it claims to do.

Reply:
Comment is still relevant:
if someone will attempt to use this function, `mvAlgo.invalidatePhysics()` expecting physics state invalidation:
1. `Flags::InWater` is recalculated at least.
2. `Flags::InAir` set to false, without care, if npc is actually in air or not
3. same for all other flags
4. cached, for ray-cast, state is still cached